constructor/utils.py

```diff
@@ -25,6 +25,13 @@
 
 from ruamel.yaml import YAML
 
+try:
+    from conda.models.version import VersionOrder
+
+    has_conda_interface = True
+except ImportError:
+    has_conda_interface = False
```

Comment:
Why would this fail? It's a dependency.

```suggestion
from conda.models.version import VersionOrder
```

Reply:
I wrote that helper function and I saw this part: https://github.com/conda/constructor/blob/d992bd7fbda7958c2d1d1d279a13727d555d7c87/constructor/conda_interface.py#L39-L46

Now that I look at the `git blame`, this was written eight years ago and is probably obsolete, so I agree that the `try` block can be removed.